jina/orchestrate/deployments/__init__.py

```diff
@@ -47,7 +49,13 @@
     from jina.serve.executors import BaseExecutor
 
 
-class Deployment(PostMixin, BaseOrchestrator):
+class DeploymentType(type(ExitStack), type(JAMLCompatible)):
```

Comment:
we do not want `ExitStack`

Reply:
Remove `ExitStack` since we are not using its features

Reply:
not possible, python will fail and say the metaclass of Deployment should be a subclass of its parent's metaclasses (which are ExitStack and JAMLCompatible)